uninstaller/uninstall-helper.ps1

```diff
@@ -4,6 +4,22 @@ for ($i=0; $i -le $args.Count - 1; $i++) {
 }
 $timeStamp = $args[-1]
 
+$subfolders = Get-ChildItem "$folder\.." -Directory -ErrorAction SilentlyContinue | ForEach-Object { $_.FullName }
+
+for ($i=0; $i -le $subfolders.Count - 1; $i++) {
+  $Source = $subfolders[$i]
+  $target = $subfolders[$i]+'\..'
```

Comment:
Lets not copy it to one level up, instead lets keep it on the same level but with random number added to folder name using `Get-Random` cmdlet.